lib/logstash/inputs/jenkins.rb

```diff
@@ -22,7 +22,7 @@ class LogStash::Inputs::Jenkins < LogStash::Inputs::Http
   # If undefined, Logstash will complain, even if codec is unused.
   default :codec, "json"
   
-  config :host, :validate => :string, :default => "jenkins.liatr.io"
+  config :host, :validate => :string, :default => "jenkins.jx.lead-dev.liatr.io"
```

Comment:
lets take these default values out since they are don't really provide any kind of meaningful default other than us being lazy about configuring the plugin

Reply:
removed default value